lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -483,6 +503,22 @@ def do_install hosts, opts = {}
           end
         end
 
+        # Helper for setting up pe_defaults & setting up the cert on the host
+        # @param [Host] host                            host to setup
+        # @param [Host] master                          the master host, for setting up the relationship
+        # @param [Array<Fixnum>] acceptable_exit_codes  The exit codes that we want to ignore
+        #
+        # @return nil
+        # @api private
+        def setup_defaults_and_config_helper_on(host, master, acceptable_exit_codes=nil)
```

Comment:
This used to run on only osx/eos - is it applicable to windows?  Does acceptable_exit_codes => nil work correctly?  Or does that indicate no acceptable exit codes?